src/main/java/core/basesyntax/vehicles/Bulldozer.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax.vehicles;
+
+import core.basesyntax.Machine;
+
+public class Bulldozer extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Bulldozer started working");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Bulldozer stopped working");
```

Comment:
Similar to the `doWork` method, the stop message should also indicate the type of machine, in this case, 'Bulldozer'. This would make the message more informative.